contributing/TESTING.md

```diff
@@ -268,16 +268,22 @@ If a Percy test flakes and you would like to trigger a rerun, you can't do that
 
 ### Running Visual Diff Tests Locally
 
-You can also run the visual tests locally during development. You must first create a free Percy account at [https://percy.io](https://percy.io), create a project, and set the `PERCY_TOKEN` environment variable using the unique value you find at `https://percy.io/<org>/<project>/integrations`. Once the environment variable is set up, you can run the AMP visual diff tests as described below.
+You can also run the visual tests locally during development. You must first create a free Percy account at [https://percy.io](https://percy.io), create a project, and set the `PERCY_TOKEN` environment variable using the unique value you find at `https://percy.io/<org>/<project>/integrations`:
+
+```sh
+export PERCY_TOKEN="<unique-percy-token>"
+```
+
+Once the environment variable is set up, you can run the AMP visual diff tests.
 
 First, build the AMP runtime and run the gulp task that invokes the visual diff script:
 
 ```sh
-gulp build
+gulp dist --fortesting
 gulp visual-diff --nobuild
 ```
```

Comment:
I think this should be split into separate commands. Lines 279-284 can be re-written as:

````
First, build the AMP runtime:

```sh
gulp dist --fortesting
```

Next, run the `gulp` task that invokes the visual diff tests:

```sh
gulp visual-diff --nobuild
```
````

Reply:
Or if you like, a single block with inline comments would work too:

```sh
# Build the AMP runtime
gulp dist --fortesting

# Run visual tests
gulp visual-diff --nobuild
```

Reply:
All set, thanks for the diligence on clarity here!